audit/gdcd/README.md

```diff
@@ -1,156 +1,157 @@
 
-The Great Docs Code Devourer (GDCD) pours through the finest hand-curated selection of MongoDB documentation pages to find
-every last morsel of our code examples. It then compares these code examples with code that it has previously devoured
-to focus on only the new code, updates to existing code, and figuring out which code has been removed from this delectable
-docs corpus. The Great Docs Code Devourer relies on a MongoDB Atlas database to savor each and every detail about these
-code examples - recording not just the code itself, but delightful metadata to make it easier to recall the code in
-novel and useful ways.
+The Great Docs Code Devourer (GDCD) processes MongoDB documentation pages to extract code examples. It compares these 
+examples with previously stored code to identify new, updated, or removed examples. GDCD stores all code examples and 
+metadata in a MongoDB Atlas database maintained by the Developer Docs team.
 
-To let it feast upon your code, or ask it to recall the finest details of its prior meals, reach out to the Developer
-Docs team.
 
-## Why devour code at all?
+Contact the Developer Docs team to use this tool or access its data.
 
-The Great Docs Code Devourer stores code examples and related metadata to a MongoDB Atlas database. This intersection of
-code and metadata enables us get interesting information about the code examples across our documentation corpus, such as:
+## Why We Devour Code
 
-- Number of code examples for a given programming language, category, product, sub-product, or keyword
+The database of devoured code examples enables powerful analysis of the documentation corpus, including:
+
+- Code example counts by programming language, category, product, sub-product, or keyword
 - Density of code examples on the page by product, sub-product, or keyword
 - Code example complexity (the intersection of the "Usage example" category and code example length)
-- Count of one-line code examples
-- List of docs projects or pages that have code examples in a given language
+- Distribution of short vs. comprehensive examples
+- Language coverage across documentation
 
-The Great Docs Code Devourer does not do any of this querying itself - it only devours code. If you want to get information
-from the musings stored by the Great Docs Code Devourer, you want the companion project,
+For querying this data, use the companion project,
 [Database of Devoured Example Code (DODEC)](https://github.com/mongodb/code-example-tooling/tree/main/audit/dodec).
 
 ## How it works
 
-The Great Docs Code Devourer uses the pipeline described below to get code examples from our documentation. We then store
-metadata about the examples, as well as the examples themselves, to a MongoDB Atlas database maintained by the Developer
-Docs team.
+GDCD follows this pipeline:
+
+1. Retrieves the latest docs project information from the Snooty Data API
+2. For each specified project, gets docs pages for current active branch
```

Comment:
Because different repos have different strategies, would it be helpful to define what "active" means here? Like, "... active branch (latest published version)".

Reply:
```suggestion
2. For each specified project, gets docs pages for current active branch (latest published version)
```